ui/address/contract/ABI/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const getNativeCoinValue = (value: unknown) => {
```

Comment:
is it specific for the contract values only or mb move it to /lib folder?

Reply:
it is used only in ABI form submit handler for parsing value of native coin value input in order to pass it to wagmi in correct format